app/src/main/res/values/strings.xml

```diff
@@ -308,8 +308,10 @@
         Final result: 'Tap [image] on the menu to capture an entire web page'. -->
     <string name="screenshot_grid_empty_text_msg_postfix">&#160;&#160;on the menu to capture\nan entire web page.</string>
 
+    <!-- Message shown when the user view the detail info of a screenshot". -->
```

Comment:
This will show "Message shown when the user view the detail info of a screenshot" as comment of <string name="screenshot_image_viewer_dialog_title">Info</string>

Maybe adding another comment before screenshot_image_viewer_dialog_title.